calicoctl/commands/node/run.go

```diff
@@ -205,6 +203,21 @@ Description:
 		// the AS number, so convert.
 		asNumber = argutils.ValidateASNumber(asNumber).String()
 	}
+	// CLI flags will take precedence over env vars
```

Comment:
@robbrockbank is correct I had to skip setting a default in the docstring so we could...

1. see if the user set a CLI flag
2. Check if there is an env var if no flag
3. Set a default if neither are present